### ISSUE TITLE
LocalCUDACluster object has no attribute 'status'

### ISSUE THREAD

**Issue by yuhan-17** [rapidsai/dask-cuda#479]
Hi, 
I'm running a multi-node multi-GPU example using cudf and cuml via dask. It hits the following error, anyone has some ideas?
My system is ubuntu 18.04, CUDA 10.2, and dask is version 0.17.
![image](https://user-images.githubusercontent.com/57814820/103328276-3680f100-4a93-11eb-882f-9f80703fd93f.png)

**Comment by pentschev** [comment]
From the error in your `pynvml.nvmlInit()` it seems that there's some issue with your CUDA install or GPU. What do you see when you run `nvidia-smi` on the command line?

**Comment by jakirkham** [comment]
> dask is version 0.17.

That's a really old version of Dask. Could you please double check that?

**Comment by pentschev** [comment]
@yuhan-17 probably meant to say dask-cuda's version is 0.17. :)

**Comment by jakirkham** [comment]
Right, but the error comes from Distributed. Hence it would be good to know what version is being used there.

**Comment by pentschev** [comment]
As per https://github.com/rapidsai/dask-cuda/issues/479#issuecomment-753890223, I don't think the error actually comes from Distributed, but simply manifests there _after_ NVML fails.

**Comment by yuhan-17** [comment]
Thank you for replying. The dask version I used was 0.17, which is not the latest. This ran in the docker image pulled from RAPIDSAI but failed due to this error. 
Then I tried conda install and following the video in multi-node multi-gpu (command line method). It worked but looked like only one GPU was used.

I think maybe more examples for multi-node multi-GPU can be provided, instead of simple ones using LocalCluster or LocalCUDACluster.
